src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -773,6 +775,14 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 		}
 	}
 
+	async enableSandbox(windowId: number | undefined, enabled: boolean): Promise<void> {
+		if (enabled) {
+			this.stateMainService.setItem('window.experimental.useSandbox', true);
```

Comment:
As discussed this leverages state main service as a workaround for not having experiments in the main process.